libs/@wingcloud/framework/src/main.ts

```diff
@@ -89,13 +89,15 @@ This is a Wing app and must be run through the Wing CLI (npm install -f winglang
   const rootId = process.env.WING_ROOT_ID;
 
   const $PlatformManager = new platform.PlatformManager({ platformPaths });
+  const $PolyconFactory = $PlatformManager.createPolyconFactory();
```

Comment:
Wondering, do we really need another global? Why not just piggy back on `$PlatformManager`?

Reply:
I don't want to expose the entire surface of $PlatformManager to all preflight files, only the dependency injector (i.e. $PolyconFactory). If there's a need to give access to other platform things then we can always revisit or change it later